docs/AboutUs.md

```diff
@@ -11,12 +11,11 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Lim Kai Xuan
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/kaixuan477.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
+[[github](https://github.com/kaixuan477)]
 [[portfolio](team/johndoe.md)]
```

Comment:
Maybe change this as well? Make a team/kaixuan477.md.

Reply:
Noted. Planned to finalise during meeting.